### ISSUE TITLE
Handling redirects in helix 

### ISSUE THREAD

**Issue by kgera** [adobe/developer.adobe.com#140]
Since helix takes over when publishing happens with helix config, so custom vcl in fastly cannot be used which is challenging when we have a large number of redirect rules. Finding an approach where there could be a way in helix to handle all redirect rules by configuration.

**Comment by filmaj** [comment]
@kgera just so I'm clear: we need a custom vcl in fastly to support various redirects for the existing .io site, is that correct? So this issue is about reconciling helix-config-based VCL with our own custom (legacy) VCL needs. Is that right?

**Comment by kgera** [comment]
@trieloff  here's the stage redirect rules list 
[StageRedirectRules.txt](https://github.com/adobe/developer.adobe.com/files/2902725/StageRedirectRules.txt)

I will put in an excel if needed in proper format.

**Comment by kgera** [comment]
@filmaj yes, we have a lot of redirect rules that need to be moved from existing dispatcher to fastly but doing that manually is a very tedious process, so i wanted to use a custom vcl. However, Lars educated me on how helix takes over the fastly config and hence, we cannot use the custom vcl which leaves us with two choices either manually adding each rule in fastly config for every redirect or just have a way where helix can use an input file format, parse it and handle all redirects,

**Comment by trieloff** [comment]
@kgera you have quite a number of Regex-based rules here. In 
https://github.com/adobe/helix-cli/issues/587 I hoped that we could get away without pattern matching. Do you think this will be a problem?

**Comment by kgera** [comment]
@trieloff  Considering our current use cases, non-existence of  pattern matching would be a problem. Eg: Scope of changes for rebranding ACP to Adobe Experience Platform would have been extremely wide in helix if we don't go with patterns, there are several pages and child pages and listing redirects in excel for each will be very tedious and a lot of manual validation would be needed.

**Comment by trieloff** [comment]
Not what I hoped for, but I think you are right.
